src/styles/images/index.md.njk

```diff
@@ -8,53 +8,70 @@ layout: layout-pane.njk
 
-The aspect ratio for photographs should be 3:2.
+Services usually work best without relying on images. Focus on writing clear, simple content for your service first.
 
-{{ example({group: "styles", item: "images", example: "default", html: false, open: false, size: "l"}) }}
+Adding images can sometimes help users better understand instructions. However, images should only be visual aids — written content must contain all the information the user needs to complete the service.
 
+If you choose to add an image to your service, you’ll need to carefully decide if the image needs to be described with alternative text (alt text).
 
-### Illustrations or representative imagery
+### Alt text
 
-If your image represents something physical, such as a letter, document or credit card you should use the aspect ratio of that item.
+Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
 
-{{ example({group: "styles", item: "images", example: "representative", html: false, open: false, size: "l"}) }}
+All images should include the alt attribute, even if it’s left empty without alt text (`alt=""`).
 
+#### When to use alt text
 
-## Alternative text
+Only add alt text when you need to describe visual details that cannot practically be included in the written content.
 
-This guidance is for government teams that build online services. [To learn about publishing alternative text in GOV.UK mainstream content, go to GOV.UK's 'Images' guidance.](https://www.gov.uk/guidance/content-design/images)
+There are some cases when an empty `alt=""` attribute is more appropriate. These include:
 
-Alternative text, or alt text, is read out by screen readers or displayed if an image does not load or if images have been switched off.
+- a decorative image that does not include important content
+- an icon with a text label, where the alt text would repeat the text label
+- an image used in a link, where the image is not needed to understand the link
+
+To decide if an image needs alt text, see the [Web Accessibility Initiative alt decision tree](https://www.w3.org/WAI/tutorials/images/decision-tree/).
 
-All images, except decorative images, must have alt text that:
+#### Writing good alt text
+
+Good alt text:
 
 - tells people what information the image provides
 - describes the content and function of the image
 - is specific, meaningful and concise
 
 Use normal punctuation, like commas and full stops, so the text is easy to read and understand.
 
-Do not:
+Generally, aim for no more than 2 sentences of alt text.
+
+Do not write alt text that:
 
-- include the name of the photographer or person who created the image
-- start with ‘Image of’, ‘Graphic of’ or ‘Photo of’
-- repeat information from the page
-- include extra information not in the image
+- includes the name of the photographer or person who created the image
+- starts with ‘Image of’, ‘Graphic of’ or ‘Photo of’
```

Comment:
feel free to ignore this, but I'd say 'photo of' is ok - it adds context to the image in a way that 'image of' and 'graphic of' does not.

Reply:
True, it might be helpful in the context of a service (as opposed to say, a news website where most images would be photos).

Reply:
@davidc-gds Any thoughts on this? I'm happy to remove 'Photo of'

```suggestion
- starts with ‘Image of’, ‘Graphic of’ or similar
```

Reply:
Or possibly:
```suggestion
- starts with ‘Image of’ or ‘Graphic of’ — try to be more specific about what kind of image it is
```

Reply:
@davidc-gds Just this last point to clarify ⬆️  — is "Photo of" useful contextual information for alt image text?